pkg/storage/endpoint/key_path.go

```diff
@@ -43,6 +42,8 @@ const (
 	keyspaceIDInfix            = "id"
 	keyspaceAllocID            = "alloc_id"
 	regionPathPrefix           = "raft/r"
+	// resource group storage endpoint has prefix `resource_group`
+	resourceGroupSettingsPath = "settings"
```

Comment:
nit: alignment problem around "=".

Reply:
According to the Go code formatter, it's separated by the comment above, so there is no need to align it with other equality signs.